package.json

```
@@ -35,14 +35,14 @@
   },
   "private": true,
   "scripts": {
-    "clean": "yarn lerna run clean --parallel",
-    "build": "yarn lerna run build",
-    "test": "yarn lerna run test --parallel",
-    "test:coverage": "yarn lerna run test:coverage --parallel",
-    "lint": "yarn lerna run lint",
-    "lint:ts:check": "yarn lerna run lint:ts:check",
-    "lint:check": "yarn lerna run lint:check",
-    "lint:fix": "yarn lerna run lint:fix --parallel",
+    "clean": "npx nx run-many --target=clean",
+    "build": "npx nx run-many --target=build",
+    "test": "npx nx run-many --target=test",
+    "lint": "npx nx run-many --target=lint",
+    "test:coverage": "npx nx run-many --target=test:coverage",
+    "lint:ts:check": "npx nx run-many --target=lint:ts:check",
+    "lint:check": "npx nx run-many --target=lint:check",
+    "lint:fix": "npx nx run-many --target=lint:fix",
```

these npx are not harming anything but are also unnecessary I can remove if we want
```suggestion
    "lint:fix": "nx run-many --target=lint:fix",
```